src/module-resolver.ts

```diff
@@ -78,6 +78,15 @@ export class LwcModuleResolverPlugin {
         )
     }
```

Comment:
I would move this to a standalone function outside of the `class` rather than exposing it on `this`. It doesn't need to be exposed externally.

Reply:
Ah I just realized we were already doing this for `isImplicitHTMLImport` as well as `isImplicitCssImport`. I think both should be moved.

AFAICT this is not exposed externally. I only see this class being used internally here:

https://github.com/salesforce/lwc-webpack-plugin/blob/97cf88784165027180469c524ad63cc9094936bf/src/index.ts#L92